nicknist/colin-and-nick-make-colors
# React initial

### What's this PR do?
Eyyyyyy. This PR adds some of the initial setup and initial boilerplate for the frontend/React side of this project. There are 'ColorPicker' components, a 'Header' component, and a 'ColorContainer' component. These add the ability to randomize and also choose a color for the five different colorpickers. In addition, I have pushed state up to the ColorContainer so all five colors are in a stateful array.

### Where should the reviewer start?
The review should pull this down, npm start and see if they are able to choose colors and randomize colors. 

### How should this be manually tested?
I would pull this down and make sure you npm install. Then npm start and check all the user interactions!

### Any background context you want to provide?
I have not implemented hooks as these components still use 'old school state.' This is something that we'll want to refactor later, as I was working with the React Color library and wanted to get it up and running first.

### Questions:
- Does the knowledge base need an update?
There could be an update of knowledge with the ColorPicker library. This is mostly used in the 'ColorPicker' component and was configured and adjusted as needed.
- Does this add new dependencies which need to be added?
I have added the basic react app dependencies that come out of the box.
